tabrepo/utils/huggingfacehub_utils.py

```diff
@@ -59,25 +60,34 @@ def upload_hugging_face(version: str, repo_id: str, override_existing_files: boo
                 token=os.getenv("HF_TOKEN"),
             )
         except Exception as e:
-            print(str(e))
+            if continue_in_case_of_error:
+                print(str(e))
+            else:
+                raise e
 
 
 def download_from_huggingface(
         version: str = "2023_11_14",
         force_download: bool = False,
         local_files_only: bool = False,
         datasets: list[str] | None = None,
+        local_dir: str | Path = None,
 ):
     """
     :param version: name of a tabrepo version such as `2023_11_14`
     :param local_files_only: whether to use local files with no internet check on the Hub
     :param force_download: forces files to be downloaded
     :param datasets: list of datasets to download, if not specified all datasets will be downloaded
+    :param local_dir: where to download local files, if not specified all files will be downloaded under
+     {tabrepo_root}/data/results
     :return:
     """
     # https://huggingface.co/datasets/Tabrepo/tabrepo/tree/main/2023_11_14/model_predictions
     api = HfApi()
-    local_dir = str(Path(__file__).parent.parent.parent / "data/results/")
+    if local_dir is None:
+        local_dir = str(Path(tabrepo.__path__[0]).parent / "data/results/"),
```

Comment:
Maybe we make a `path_data` constant that we can import that is equivalent to this? Then it is easy to check which logic in TabRepo relates to a given output artifact directory. I think we had something similar for the paper scripts

Reply:
Ok I did this proposed change.

Reply:
Was the change pushed? I don't see it

Reply:
https://github.com/autogluon/tabrepo/pull/96/files#diff-51bfa356430ae1cbb13d712982449a39196b3a8d1932d2bd0be54492d600dabcR87